backend/fms_core/models/user_profile.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.conf import settings
+
+from fms_core.schema_validators import JsonSchemaValidator
+from fms_core.models.freezeman_user import FreezemanUser
+
+PREFERENCES_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "fms:preference",
+    "title": "Preference schema",
+    "description": "Schema used to define user's preferences.",
+    "type": "object",
+    "properties": {
+        "table.sample.page-limit": {"type": "number"}
+    }
+}
+
+class UserProfile(models.Model):
+    user = models.ForeignKey(FreezemanUser, on_delete=models.PROTECT, related_name="user_profile")
+    preferences = models.JSONField(
+        default={
```

Comment:
don't hardcode specific preferences in the default json. This is likely to change overtime and we won't know what is default and what is not ... Leave it null if it is not defined ... We can override this at a higher level (within a service or viewset) to return the frontend a default set of preferences that can be updated without migration.